ui/lib/core/addon/templates/components/replication-header.hbs

```diff
@@ -25,8 +25,8 @@
           <span class="sep">
             /
           </span>
-          {{#link-to "vault.cluster.replication-dr-promote"}}
-            Disaster Recovery
+          {{#link-to "vault.cluster.replication.index"}}
```

Comment:
The only time a breadcrumb shows on the route `ui/vault/replication-dr-promote/details` or the `ui/vault/replication-dr-promote/` is when there is no DR Secondary and you land on the empty state component.  In this case. you can use the breadcrumb to navigate back to the replication index page which lets you setup replication.  See image here: 
![image](https://user-images.githubusercontent.com/6618863/86620365-70a48680-bf79-11ea-9c54-f413434c041e.png)